src/lizaalert/courses/models.py

```diff
@@ -438,16 +439,19 @@ class Subscription(TimeStampedModel):
 
     user - ForeignKey на модель user
     course - ForeignKey на модель course.
-    enabled - признак активности записи на курс.
+    status - признак активности записи на курс.
     """
 
     class Status(models.TextChoices):
-        ENROLLED = "enrolled", "Запись активна"
+        ENROLLED = "enrolled", "Записан на курс"
         NOT_ENROLLED = "not_enrolled", "Запись не активна"
+        IN_PROGRESS = "in_progress", "Курс проходится"
+        AVAILABLE = "available", "Курс доступен"
+        COMPLETED = "completed", "Курс пройден"
 
-    user = models.ForeignKey(User, on_delete=models.PROTECT, related_name="student")
-    course = models.ForeignKey(Course, on_delete=models.PROTECT, related_name="course")
-    enabled = models.CharField(
+    user = models.ForeignKey(User, on_delete=models.PROTECT, related_name="subscriptions")
+    course = models.ForeignKey(Course, on_delete=models.PROTECT, related_name="subscription")
```

Comment:
related_name  во множественном числе нужно использовать, но здесь лучше не получать список подписок для курса (для этого лучше явно запрос написать)

```suggestion
    course = models.ForeignKey(Course, on_delete=models.PROTECT, related_name="+")
```